src/Destination/Adapter/RestApiOutput.php

```diff
@@ -65,17 +66,17 @@ public function write($values)
     private function call($config, $values = [])
     {
         $body = null;
-        $returnConfig = (isset($config['return']))? $config['return'] : [];
+        $returnConfig = (isset($config['return'])) ? $config['return'] : [];
 
         if (isset($config['body'])) {
             $body = MappingHelper::parseContent($config['body'], $values);
         }
 
-        if (! isset($config['bodyType']) || $config['bodyType'] == 'json') {
+        if (!isset($config['bodyType']) || $config['bodyType'] == 'json') {
             $body = json_encode($body);
         }
 
-        $headers    = MappingHelper::parseContent($config['headers'], $values);
+        $headers = MappingHelper::parseContent($config['headers'], $values);
```

Comment:
Avoid using static access to class '\Dealweb\Integrator\Helper\MappingHelper' in method 'call'.